programs/test/CMakeLists.txt

```diff
@@ -13,8 +13,10 @@ endif(ENABLE_ZLIB_SUPPORT)
 add_executable(selftest selftest.c)
 target_link_libraries(selftest ${libs})
 
-add_executable(benchmark benchmark.c)
-target_link_libraries(benchmark ${libs})
+if(!ENABLE_PSA)
```

Comment:
Again, I don't think this is necessary.

Reply:
Benchmark is only testing crypto. If the crypto submodule is enabled and used, why keep the top level benchmark program and not just use the crypto submodule one instead?

Reply:
Because it's the Mbed TLS program using the alternative backend. I think enabling the submodule should change only the crypto library, nothing else. If it does before we move the files into the submodule it will be confusing.